docs/military/darwin.md

```diff
@@ -4,4 +4,189 @@
+
+Issue `F180` or `RFL`; whichever is lower, to aircraft cleared via an RNAV SID or aircraft with a MTOW of 25000kg or more  
+Issue `F180` or `RFL`; whichever is lower, to jet aircraft not cleared via an RNAV SID regardless of MTOW  
+Issue `A030` or `RFL`; whichever is lower, to aircraft not cleared via an RNAV SID with a MTOW of less than 25000kg  
+
+
+#### VFR Departures
+`A020` or the planned level; whichever is lower, to all VFR aircraft.  
+
+VFR aircraft are required to track via one of the published VFR Routes.  
+
+VFR routes shall be assigned based on the destination radial from Darwin.  
+
+Assign VFR routes in accordance with the following radial chart:  
+
+|Outbound Radial |Assigned VFR Route|
+|---|---|
+|360 – 040 |VFR Route 1|
+|041 – 084 |VFR Route 2|
+|085 – 124 |VFR Route 3|
+|125 – 180 |VFR Route 4|
+|181 – 224 |VFR Route 5|
+|225 – 359| Direct|
+
+!!! tip
+    If a VFR aircraft has not planned via a VFR route as above, use the phraseology: “ABC, cleared amended route VFR route 1, maintain A020, squawk 4512”
+
+
+#### Auto-Release Wedge
+
+The Darwin auto-release wedges extend from the departure end runway threshold, 30 degrees splayed from the centreline, to 7nm  
+
+Auto-release at Darwin is only available for VFR aircraft and low-level IFR aircraft that conform with the TMP  
+
+Tower may depart aircraft that conform with the above conditions within the auto-release wedge using the following headings:
+
+|Departure Type| Runway 11| Runway 29|
+|---|---|----|
+|VFR |H070V, H106V or H140V| H260V, H286V or H320V|
+|IFR (Visual Dep)| H070V, H106V, or H140V |H260V, H286V or H320V|
+|IFR (Radar)| H070, H106 or H140 |H260, H286 or H320|
+
+!!! tip
+    Tower shall provide visual separation between aircraft in the auto-release wedge and subsequent departures until the preceding departure has been maneuvered by TCU
+
+#### Non-RNAV Departures
+For non-RNAV approved IFR aircraft with a MTOW of 25000kg or more, issue a Radar SID  
+For non-RNAV approved IFR aircraft with a MTOW of less than 25000kg, issue a visual departure or Radar SID  
+## Coordination
+### DN TCU / ENR
+#### Departures
+Voiceless coordination is in place from DN TCU to TRT for aircraft:  
+Planned at or above `F180`: `Assigned F180`  
+Planned below `F180`: `Assigned the RFL`  
+
+Any aircraft not meeting the above criteria must be prior coordinated to ENR.
+
+!!! example
+    <span class="hotline">**DN TCU** -> **TRT**</span>: "PFY1234, with your concurrence, will be assigned F160, for my separation with JTE654"  
+    <span class="hotline">**TRT** -> **DN TCU**</span>: "PFY1234, concur F160"  
+
+#### Arrivals
+The Standard assignable level from TRT to DN TCU is `A100`, and assigned a STAR. All other aircraft must be prior coordinated
+
+### DN TCU / DN ADC
+#### Auto Release
+
+Auto Release shall be used for aircraft that are:    
+a) Departing from a runway nominated on the ATIS; and  
+b) Issued a Procedural SID; and   
+c) Assigned `F180`.
```

Comment:
Confirm is F180 correct? Should it be like A060 or something?

Reply:
Its F180 due to the TMP